jwst/tests_nightly/general/miri/test_srctype2.py

```diff
@@ -27,7 +27,7 @@ def test_srctype2(_bigdata):
 
 
     SourceTypeStep.call(_bigdata+'/miri/test_srctype/jw80600012001_02101_00003_mirimage_flat_field.fits',
-                        output_file=output_file_base
+                        output_file=output_file_base, suffix='sourcetypestep'
```

Comment:
Curious as to why you're including "step" in this suffix, when all the other tests don't.

Reply:
Because that's what the previous test used for the output file name.  I didn't want to change things too much.  But for consistency sake, perhaps this would be good thing to change in an upcoming PR.

Reply:
👍